.github/workflows/build-test-deploy.yml

```diff
@@ -31,9 +31,23 @@ permissions:
   security-events: write
 
 jobs:
+  paths-ignore:
+    runs-on: ubuntu-latest
+    outputs:
+      skip: ${{ steps.paths-ignore.outputs.skip }}
+    steps:
+      - name: Skip job when only Markdown files are changed
+        uses: kunitsucom/github-actions-paths-ignore-alternative@v0.0.4
+        id: paths-ignore
+        with:
+          paths-ignore: |-
+            ^.*\.md$
```

Comment:
Will this also skip if there was a markdown file added along with new code in the commit? I don't really see the `only markdown files` part